templates/theme/styles/base.less

```diff
@@ -277,6 +279,7 @@ atom-text-editor .search-results .marker.current-result .region,
     font-style: italic;
   }
 
+  &.heading,
   &.heading .punctuation.definition.heading {
```

Comment:
When doing a [search](https://github.com/search?q=user%3Aatom+.punctuation.definition.heading&type=Code&utf8=%E2%9C%93) I can only find it this file: https://github.com/atom/language-coffee-script/blob/master/grammars/coffeescript%20(literate).cson#L131. Seems like when Markdown is used in a CoffeeScript file? History doesn't reveal much either.

> Just remove it, and hope nobody notices anything?

Yeah, I would say so. It sounds really specific for a "base" selector.